templates/slave_etc_named.conf.j2

```diff
@@ -24,6 +24,7 @@ options {
 {% endif %}
 
   recursion {% if bind_recursion %}yes{% else %}no{% endif %};
+  {% if bind_recursion %}allow-recursion { {{ bind_allow_recursion|join('; ') }}; };
 {% if bind_forwarders|length > 0 %}forwarders { {{ bind_forwarders|join('; ') }}; };{% endif %}
```

Comment:
There is no matching endif clause for this line, causing a templating failure.

Reply:
Ah rats. I forgot to copy that line as I only did a quick and dirty web ide change. It's fixed now.